osgar/replay.py

```diff
@@ -42,20 +37,28 @@ def replay(args, application=None):
                 names.append(edge_from)
             inputs[1 + names.index(edge_from)] = edge_to.split('.')[1]
 
-    # start reading log from the beginning again
+    print("inputs:")
```

Comment:
I am wondering if you use conditional listing - because mostly I am just checking that everything runs (without output) and now it is easy to overlook:
```
original args: b"['/home/robot/git/osgar/subt/__main__.py', 'run', 'config/kloubak2-subt-estop-lora.json', '--note', 'test dropdowns na hale', '--side', 'right']"
streams:
  1 app.desired_speed
  2 app.pose2d
  3 app.artf_xyz
  4 app.pose3d
  5 app.stdout
  6 app.request_origin
  7 app.bumpers_front
  8 app.bumpers_rear
  9 detector.artf
 10 detector.dropped
 11 detector.debug_artf
 12 detector.stdout
 13 reporter.artf_cmd
 14 kloubak.pose2d
 15 kloubak.emergency_stop
 16 kloubak.encoders
 17 kloubak.can
 18 kloubak.bumpers_front
 19 kloubak.bumpers_rear
 20 kloubak.downdrops_front
 21 kloubak.downdrops_rear
 22 can.can
 23 can.raw
 24 serial.raw
 25 lidar.raw
 26 lidar.scan
 27 lidar_tcp.raw
 28 lidar_back.raw
 29 lidar_back.scan
 30 lidar_back_tcp.raw
 31 camera.raw
 32 camera_back.raw
 33 imu.orientation
 34 imu.rotation
 35 imu_serial.raw
 36 estop.emergency_stop
 37 estop.raw
 38 estop_serial.raw
 39 lora.raw
 40 lora.cmd
 41 lora.robot_status
 42 lora_serial.raw
inputs:
  1 slot_desired_speed
 22 slot_can
outputs:
 14 pose2d
 15 emergency_stop
 16 encoders
 17 can
 18 bumpers_front
 19 bumpers_rear
 20 downdrops_front
 21 downdrops_rear
Emergency STOP: True
```
... I would kind of vote for the command line but not the complete listing as default??

Reply:
Sure, no problem.